site/source/docs/porting/Debugging.rst

```diff
@@ -137,6 +137,45 @@ Debug printouts can even execute arbitrary JavaScript. For example::
   }
 
 
+Printing exception messages
+===========================
+
+Exceptions are thrown from WebAssembly using exception pointers, which means that try/catch/finally blocks in JavaScript will only receive a number, which represents a pointer into linear memory. In order to get the exception message, the user will need to create some WASM code which will extract the meaning from the exception, for example:
```

Comment:
Would you mind wrapping these paragraphs at 80 chars?  I know that other documentation mostly doesn't do this but  I'm trying to move in that direction.

Reply:
done.